proto/tpm_enrollz.proto

```diff
@@ -81,6 +83,25 @@ message RotateOIakCertResponse {
   // changes.
 }
 
+// The RotateAIKCertRequest handles the workflow for enrollment of TPM1.2
+// devices. The initial request will include the issuer_public_key to allow
+// the building of the AIK which will then be returned and used to generate
+// the AIK cert. For any updates after initial enrollment the new AIK cert
+// can just be returned.
+message RotateAIKCertRequest {
+  oneof value {
+    bytes issuer_public_key = 1;
+    string aik_cert = 2;
+  }
+}
+
+message RotateAIKCertResponse {
+  oneof value {
+    bytes application_identity_public_key = 1;
+    bool finalized = 2;
```

Comment:
Instead of an boolean, we can return the AIK certificate which was extracted from the cert payload. The issuer can validate that the extracted AIK certificate is the intended one to this device. RotateAIKCertReponse can be

```
message RotateAIKCertResponse {
  oneof value {
     bytes application_identity_request = 1;
     string aik_cert = 2;
  }
}
```

Reply:
not sure that brings much value as with being part of the same stream... how would the device have done anything else?